tensorrt_llm/_torch/pyexecutor/py_executor.py

```
@@ -1653,6 +1682,13 @@ def _enqueue_responses(self, responses: Iterable[Tuple[int, LlmResponse]]):
                         self.responses[req_id].append(resp)
                     else:
                         self.responses.update({req_id: [resp]})
+                    # (TODO: joyang) There are other types of responses, we need to sort out.
+                    if type(
+                            resp
```

Ruff (0.12.2)

Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)